chat-plugins/modified.js

```diff
@@ -291,16 +248,16 @@ exports.commands = {
 		} else if (nextGroup in Config.groups && currentGroup in Config.groups && Config.groups[nextGroup].rank < Config.groups[currentGroup].rank) {
 			if (targetUser && room.users[targetUser.userid] && !Config.groups[nextGroup].modlog) {
 				// if the user can't see the demotion message (i.e. rank < %), it is shown in the chat
-				targetUser.send(">" + room.id + "\n(You were demoted to Room " + groupName + " by " + user.name + ".)");
+				targetUser.send(">" + room.title + "\n(You were demoted to Room " + groupName + " by " + user.name + ".)");
```

Comment:
What does this improve? It might actually break something because were sending to client with `targetUser.send`. Stick with `room.id` in this case.